### PR TITLE
Sentry config setup

### PR DESCRIPTION
**Pull request by conbrad**
- Sets up sentry for errors, metrics, replays and user feedback
# Test Links:
[Landing Page](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3557.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Comment by brettedw** [comment]
The `Report a Bug` button sits over top of our scale bar in ASA and goes a touch into the Morecast grid, not sure if it effects any other apps

File: web/src/components/Contact.tsx [hunk 1]
```diff
@@ -28,17 +29,10 @@ interface Props {
 
 const Contact = (props: Props) => {
   const productName = props.productName
-
+  console.log(productName)
```
Comment (dgboss) [inline web/src/components/Contact.tsx:32]: Can this be removed?

File: web/src/types/window.d.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // use TypeScript's interface declaration merging
 interface Window {
   env: {
+    [x: string]: string
```
Comment (dgboss) [inline web/src/types/window.d.ts:4]: Is this necessary?

Comment (conbrad) [reply web/src/types/window.d.ts]: Thanks, accidental, fixed in: https://github.com/bcgov/wps/pull/3557/commits/df7c95d01c40578f4652f107f2557febb581bd4c

File: web/src/components/Contact.tsx (hunk 1 above)
Comment (conbrad) [reply web/src/components/Contact.tsx]: Removed in: https://github.com/bcgov/wps/pull/3557/commits/df7c95d01c40578f4652f107f2557febb581bd4c

File: web/src/components/Contact.tsx [hunk 3]
```diff
@@ -22,23 +23,10 @@ const Root = styled('div')({
...
 })
 
-interface Props {
-  productName: string
-}
-
-const Contact = (props: Props) => {
-  const productName = props.productName
-
+const Contact = () => {
```
Comment (dgboss) [inline web/src/components/Contact.tsx:26]: Can we get rid of this `Contact` component entirely and just import the `FeedbackButton` into the `GeneralHeader` component? Or if you want to keep the `Contact` component there is no need for the `Root` styled div (can be deleted along with the classes.

File: web/src/features/landingPage/components/Sidebar.tsx [hunk 4]
```diff
@@ -235,8 +236,8 @@ export const Sidebar: React.FunctionComponent = () => {
         {renderLargeCollaborate()}
         <Subheading title="Support" />
         <Content>
-          <Typography>To report bugs or receive support on technical issues, please email:</Typography>
-          <Email href={'mailto:BCWS.PredictiveServices@gov.bc.ca'}>BCWS.PredictiveServices@gov.bc.ca</Email>
+          <Typography>To report bugs or receive support on technical issues, please click below:</Typography>
+          <FeedbackButton color="primary" />
```
Comment (dgboss) [inline web/src/features/landingPage/components/Sidebar.tsx:240]: I think we need a little white space above the button.

nit: What do you think about centering the button horizontally?

**Review by dgboss — APPROVED** [review]
Great work! Just a few trivial questions.

**Comment by dgboss** [comment]
For a future enhancement it would be great if we could prepopulate the form with the logged in user's email address and idir or name.

**Comment by conbrad** [comment]
> For a future enhancement it would be great if we could prepopulate the form with the logged in user's email address and idir or name.

Yes! We can, I'll see if it's quick and easy or more involved -- if the former I'll add it in.

**Review by brettedw — APPROVED** [review]
Love it!

**Comment by conbrad** [comment]
Closing until we want to revisit the feedback form

**Pull request closed without merging**